CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 ## changelog
 
+## Unreleased
```

Comment:
Probably makes the most sense to call this v`2.2.0` when released 👍

Reply:
I'm wondering if 3.0.0 might make more sense since an existing bring-your-own-client implementation might break with the needle upgrade since needle has different options than request 🤔

Reply:
Great point, hadn't considered that. 👍 to `3.0.0`